app.js

```diff
@@ -1,98 +1,307 @@
+  let startDate = data.labels
+    ?.filter((fields) => {
+      return fields.labelTypeId === "83c1c22a-6776-0ad6-68d2-819e1c6eec66";
+    })
+    .pop()?.validity?.start;
+  let activeState = data.labels?.filter((fields) => {
+    return (
+      fields.labelTypeId === "83c1c22a-6776-0ad6-68d2-819e1c6eec66" &&
+      !fields.validity.hasOwnProperty("end")
+    );
+  })
+    ? ORGANIZATION_STATUS.ACTIVE
+    : ORGANIZATION_STATUS.INACTIVE;
+  let email = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "f611e91d-a811-4519-afa6-08f0e9dcdd9b" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+  let rechtsvorm = data.organisationClassifications
+    ?.filter((fields) => {
+      return (
+        fields.organisationClassificationTypeId ===
+          "9bae7539-64fd-5759-743c-ad9dfa4143d4" ||
+        fields.organisationClassificationTypeId ===
+          "1131205e-9212-435d-b4cd-b0d955d08bcf"
+      );
+    })
+    .pop()?.organisationClassificationName;
+  let phone = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "d792d4ff-8c34-4336-8434-56ecb40f2fa4" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+  let website = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "6763f372-02c5-478c-b7d7-802dc60a64b9" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+  let province = data.parents?.parentOrganisationName;
+  let formattedAddress = data.locations
+    ?.filter((fields) => {
+      return (
+        fields.isMainLocation === true &&
+        !fields.validity?.hasOwnProperty("end")
+      );
+    })
+    .pop()?.formattedAddress;
+
+  //main location according to KBO
+  let adressComponent = data.locations
+    ?.filter((fields) => {
+      return (
+        fields.locationTypeId === "537c0b5b-8ab8-fc3d-0b37-f8249cbdd3ba" &&
+        !fields.validity?.hasOwnProperty("end")
+      );
+    })
+    .pop()?.components;
+
+  return {
+    changeTime: changeTime ? changeTime : " ",
+    organisationName: organisationName
+      ? organisationName
+      : "Geen gegevens opgenomen in KBO",
+    shortName: shortName ? shortName : organisationName,
+    ovoNumber: ovoNumber ? ovoNumber : "Geen gegevens opgenomen in KBO",
+    kboNumber: kboNumber ? kboNumber : "Geen gegevens opgenomen in KBO",
+    formalName: formalName ? formalName : "Geen gegevens opgenomen in KBO",
+    startDate: startDate ? startDate : "Geen gegevens opgenomen in KBO",
+    activeState: activeState,
+    rechtsvorm: rechtsvorm ? rechtsvorm : "Geen gegevens opgenomen in KBO",
+    email: email ? email : "Geen gegevens opgenomen in KBO",
+    phone: phone ? phone : "Geen gegevens opgenomen in KBO",
+    website: website ? website : "Geen gegevens opgenomen in KBO",
+    province: province ? province : "Geen gegevens opgenomen in KBO",
```

Comment:
**Suggestion:** Use the [Nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) will increase readability. 

```suggestion
    province: province ?? "Geen gegevens opgenomen in KBO",
```